pegasus/sites.v3/code.org/views/header2.haml

```diff
@@ -39,6 +39,9 @@
             %a.headerlink{:href=>"/help"}= I18n.t('header_help_us')
             %a.headerlink{:href=>"/about"}= I18n.t('header_about')
             %a.headerlink{:href=>CDO.studio_url("/projects/public")}= I18n.t('header_project_gallery')
+          - else
+            %a.headerlink{:href=>"/courses"}=I18n.t('nav.header.courses')
```

Comment:
Do these work?  i think "nav.header." prefix is for dashboard strings, not pegasus.